test/scripts/e2e-kind.sh

```
@@ -215,7 +215,7 @@ ginkgo --nodes=${NUM_NODES} \
 	--flake-attempts=${FLAKE_ATTEMPTS} \
 	/usr/local/bin/e2e.test \
 	-- \
-	--kubeconfig=${HOME}/ovn.conf \
+	--kubeconfig=${HOME}/admin.conf \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)